tmt/steps/discover/fmf.py

```diff
@@ -43,26 +43,43 @@ class DiscoverFmfStepData(tmt.steps.discover.DiscoverStepData):
         default=cast(Optional[str], None),
         option=('-u', '--url'),
         metavar='REPOSITORY',
-        help='URL of the git repository with fmf metadata.')
+        help="""
+            Git repository containing the metadata tree.
+            Current git repository used by default.
+            """)
     ref: Optional[str] = field(
         default=cast(Optional[str], None),
         option=('-r', '--ref'),
         metavar='REVISION',
-        help='Branch, tag or commit specifying the git revision.',
+        help="""
+            Branch, tag or commit specifying the desired git
+            revision. Defaults to the remote repository's default
+            branch if ``url`` was set or to the current ``HEAD``
+            of the current repository.
+            """,
```

Comment:
This does not cover all `ref` details from the specification:

* https://tmt--3311.org.readthedocs.build/en/3311/spec/plans.html#fmf

Plus, would expect that we remove the duplicate information from the original place (e.g. from the spec) completely otherwise we're not actually deduplicating, right?

Reply:
@psss good catch! `ref` updated, about removing the information I agree, but I wasn't sure (asked in my second comment) Including it in this PR

Reply:
@psss Specs duplicated info removed !

Reply:
@psss are we good here?

Reply:
Yes, thanks for moving the stuff!